mewarpx/tests/test_particle_diag.py

```diff
@@ -0,0 +1,106 @@
+"""
+Test that verifies data is plotted correctly from the
+Monte-Carlo Collision script based on case 1 from
+Turner et al. (2013) - https://doi.org/10.1063/1.4775084
+"""
+
+from examples.capacitive_discharge import PLASMA_DENSITY
```

Comment:
Let's avoid grabbing code from something not designed to be a module, so figure out how to eliminate this line.